### PR TITLE
ADV3 -- add advance setting tab

### PR DESCRIPTION
**Pull request by ermiaSparkle**
Transfer the advance setting to tabs

### CODE REVIEW

**Review by denisdimitrov — CHANGES_REQUESTED** [review]
This is not adding the navigation bar, this just moves the advanced settings inside 'manage space' screen. 

You need to wrap the screen with <WithNavigationbar/> to add the navigation bar.

File: src/components/organisms/AdminVenues/AdminVenues.scss [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 .admin-venue {
   width: 100%;
-  height: 100%;
+  padding: 8%;
```
Comment (denisdimitrov) [inline src/components/organisms/AdminVenues/AdminVenues.scss:5]: All of our paddings are with pixel values, please keep it consistent.

File: src/components/organisms/AdminVenues/AdminVenues.scss [hunk 2]
```diff
@@ -15,6 +16,24 @@
     font-weight: bolder;
   }
 
+  &__button {
+    height: 45px;
+    background-color: $primary;
+    padding: 1rem;
```
Comment (denisdimitrov) [inline src/components/organisms/AdminVenues/AdminVenues.scss:22]: There is `%` `px` and `rem` in a single file for different padding values. Very confusing.

File: src/components/organisms/AdminTabsBar/AdminTabsBar.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+export const AdminTabsBar: React.FC<AdminTabsBarProps> = ({
+  activeNav = "",
+  children,
+}) => {
+  const { venueId } = useParams<AdminNavBarRouteParams>();
+
+  const adminTabsUrl = useCallback(
+    (key: string) => {
+      return key === AdminTabsBarTab.advanceSetting
+        ? adminNGSettingsUrl(venueId)
```
Comment (denisdimitrov) [inline src/components/organisms/AdminTabsBar/AdminTabsBar.tsx:56]: It should be using the same route, not another one.
If I can access the spaces like this:
https://co-reality-staging--preview-pr-1991-4osrrjuw.web.app/admin-ng/venue/8mars/spaces
Then I should be able to access advanced settings like this:
https://co-reality-staging--preview-pr-1991-4osrrjuw.web.app/admin-ng/venue/8mars/advanced

In the current implementation it's another route which is incorrect.

Ideally you should show this to Sofi and have a conversation about it. IMO only the "Advanced" Should be moved here and the other 2 tabs should exist in the other route.

**Review by denisdimitrov — CHANGES_REQUESTED** [review]

**Comment by ermiaSparkle** [comment]
Duplicate of #2268

**Pull request closed without merging**